### DISCUSSION TITLE
Console output of agents (console logging) - how to silence it?

### DISCUSSION THREAD

**Discussion by skye0402** [crewAIInc/crewAI#1225] [Q&A]
I found that when setting `verbose` to False for both agents and crew that makes the thoughts and handover logs go away but e.g. all output from tools are shown in full verbosity: Web site content etc.

How can I stop this? It's jamming my applications log and it's hard to keep an overview.

**Comment by yogeshs-vaf** [comment]
Did you find it?

**Comment by leobenkel** [comment]
how to silence prompt and sensitive information ?

**Comment by matheusm-luz** [comment]
I don't know if it could be the same problem I had. But even setting `verbose=False` for my agents, the logs were triggered. To disable event logging, before running my agent, I set it in the global scope of the class:
```
from crewai.crew import EventListener
from crewai.utilities.events.event_listener import ConsoleFormatter

my_listener = EventListener()
my_listener.formatter = ConsoleFormatter(verbose=False)

[...]
```

If it's the same problem as your's, maybe this doc helps: https://docs.crewai.com/en/concepts/event-listener#for-crew-based-applications
